composer.json

```diff
@@ -6,7 +6,7 @@
         "php": ">=7.1",
         "humanmade/hm-gtm": "~2.0.7",
         "altis/aws-analytics": "~4.6.0",
-        "altis/analytics-integration-segment": "~0.1.0"
+        "altis/analytics-integration-segment": "~0.2.0"
```

Comment:
For a 0.x version the caret version range has the same effect as the tilde, no need to change it, just a fun fact

Reply:
Do you mean dependabot should've picked it up? Because I forced an update and it didn't.